components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/utils/MappingUtil.java

```diff
@@ -543,9 +549,20 @@ public static int mapSecuritySchemeListToInt(List<String> securityScheme) {
                 default:break;
             }
         }
+        
+        if (securityScheme.size() == 0) {
+            securitySchemeValue = 3;
```

Comment:
Since this is just a temporary fix shall we make testing both schemes easy by setting 3? Otherwise, we will always have to use REST API for testing API Key till the UI is added. We will have to make sure to set this to "1" (or what is appropriate) after the UI is fixed.